recipes/pvcaptest/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "0.8.0" %}
+
+package:
+  name: pvcaptest
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/c/captest/captest-{{ version }}.tar.gz
+  sha256: 45d973484208cef9dfb0e35d9e7caeb95257d71736dcf6d9e292481a6dc345c5
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - bokeh >=1
+    - matplotlib >=2
+    - numpy >=1.13.0
+    - pandas >=0.24
+    - pip
+    - setuptools
+    - python >=3.5
+    - python-dateutil >=2.5
+    - scikit-learn >=0.19
+    - statsmodels >=0.8
+    - holoviews >=1.11
+    - pvlib-python >=0.7
+
+  run:
+    - bokeh >=1
+    - matplotlib >=2
```

Comment:
```suggestion
    - matplotlib-base >=2
```

Reply:
I made this change to matplotlib-base.  What is the difference between these two?

Reply:
If you do not need the ``qt`` backend just ``matplotlib-base`` is sufficient 
``matplotlib`` will pull ``pyqt`` and the ``qt`` stack, which are a quite large dependency